cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/store/preview/DefaultPreviewStore.java

```diff
@@ -388,6 +409,24 @@ public byte[] getPreviewRequestPollerInfo(ApplicationId applicationId) {
     return null;
   }
 
+  public ApplicationId getApplicationIdForPollerInfo(byte[] pollerInfo) {
```

Comment:
nit: can be renamed `public ApplicationId getApplicationId(byte[] pollerInfo)` to eliminate redundancy in naming.